jobs/godojo/templates/config/dojoConfig.yml.erb

```diff
@@ -0,0 +1,160 @@
+# Default config for godojo installations
+#
+# How to read the lines below:
+# Each line represents a value used by the installer in this format:
+# [name]: [default] # [ENV] - [Description]
+# where
+# [name] is the name of the configuration item
+# [default] is the default value for the configuration item
+# [ENV] is the environmental variable used to override the config item at run time
+# [Description] is a description of that the config item's purpose
+#
+# If the following values are left empty aka "", they will be randomly generated at install time
+# CredentialAES256Key
+# SecretKey
+
+Install:
+  Version: "<%= p('dojo.install.version') %>" # DD_Version - Release version of DefectDojo from Github Releases
+  SourceInstall: true # DD_SourceInstall - Boolean if a source install is desired (vs a release)
```

Comment:
Aren't we installing it from the blob, not from GitHub?

Reply:
Godojo gets installed from a blob, but the actual DefectDojo install is from Github